mtgjson5/classes/mtgjson_set.py

```diff
@@ -19,6 +19,7 @@ class MtgjsonSetObject:
     cards: List[MtgjsonCardObject]
     code: str
     code_v3: str
+    foreign_printed_languages: List[str]
```

Comment:
I'm really thinking we want to support English too... As some sets in the future will be non-english, and then it'll get confusing. Idk, just thinking right now. Might want to just change this to printed_languages.